content/influxdb/cloud-iox/reference/internals/storage-limits.md

```diff
@@ -0,0 +1,64 @@
+---
+title: IOx storage engine limits
+description: >
+  The InfluxDB IOx storage enforces specific limits on the storage level.
+menu:
+  influxdb_cloud_iox:
+    name: IOx storage limits
+    parent: InfluxDB Cloud internals
+weight: 201
+---
+
+The InfluxDB IOx storage enforces specific limits on the storage level.
+
+- [Terminology](#terminology)
+- [Service-level limits](#service-level-limits)
+- [Error messages](#error-messages)
+
+## Terminology
+
+- **namespace**: organization+bucket
+- **table**: [measurement](/influxdb/cloud-iox/reference/glossary/#measurement)
+- **column**: time, tags and fields are structured as columns
+
+## Service-level limits
+
+The IOx storage engine enforces the following storage-level limits:
+
+- **Maximum number of tables per namespace**: 500
+- **Maximum number of columns per table**: 200
```

Comment:
I have just confirmed this has been deployed now - **new** namespaces will use this lower limit (previously 10,000)